go/chat/storage/outbox.go

```diff
@@ -351,27 +351,43 @@ func (o *Outbox) RetryMessage(ctx context.Context, obid chat1.OutboxID,
 	return res, nil
 }
 
-func (o *Outbox) UpdateMessage(ctx context.Context, replaceobr chat1.OutboxRecord) error {
```

Comment:
revert these changes? seem unused

Reply:
I kind of like them, I reverted the changes to `MessageDeliverer` though.